pipelines/installing-pipelines.adoc

```diff
@@ -13,10 +13,8 @@ toc::[]
 * You have access to an {product-title} cluster using an account with `cluster-admin` permissions.
 
 * You have installed `oc` CLI.
-
-////
 * You have installed xref:../cli_reference/tkn_cli/installing-tkn.adoc[OpenShift Pipelines (`tkn`) CLI] on your local system.
```

Comment:
@Preeticp this is missing an anchor id.

Reply:
Ack, will fix, thank you

Reply:
Fixed @vikram-redhat will squash and keep it ready to be merged .